resources/red/ranges/defensive ranges.md

```diff
@@ -0,0 +1,771 @@
+# Red Glitchless Defensive Ranges
+## **MP - IT - Blizzard - Agatha Candy** Route
+### Notes:
+ - `move`* denotes a badge boost, `move`** denotes 2, etc.
+ - `move`@-1 denotes a defensive stat drop, `move`@-2 denotes 2, etc.
+ - [`rolls`] denotes a critical hit
+   - Note : there are no [`rolls`] for stat changes because critical hits ignore all of them
+ - All rolls include 1/39 max roll
+
+## Squirtle
+
+**Rival** _TODO_: this is not accurate, needs to be fixed
+
+Bulbasaur    | Tackle
+------------ | -------------
+&nbsp;       | 2-3 [3-5]
```

Comment:
Github doesn't seem to recognise this symbol in the diff. Is this intended behaviour?

Reply:
i think so, its displayed only in the source

Reply:
yeah its a non-breaking-space